.github/workflows/alpha-release.yml

```diff
@@ -0,0 +1,79 @@
+name: Manual Alpha Package Release
+
+on:
+  # Allows running workflow manually from the GitHub Actions tab
+  workflow_dispatch:
+
+jobs:
+  build_and_test:
+    name: Build and Test
+    runs-on: ubuntu-latest
+
+    steps:
+      # Check-out repo
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      # Ensure node version is great enough
+      - name: Use Node.js v12.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+
+      # Install dependencies
+      - name: Install rush
+        run: npm install -g @microsoft/rush
+      - name: Install dependencies
+        run: rush install
+
+      # Get datetime
+      - name: Get datetime for alpha release name
+        id: datetime
+        run: |
+          echo "::set-output name=datetime::$(date +'%Y%m%d.%H%M')"
+
+      # Bump alpha package versions
+      - name: Bump alpha package versions
+        run: node common/config/node_modules/beachball/bin/beachball canary --canary-name alpha-${{ steps.datetime.outputs.datetime }} --tag dev --no-publish
+
+      # Build packages
+      - name: Build @azure/communication-react package
+        run: rush build -t @azure/communication-react
+
+      # Test Packages
+      - name: Test @azure/communication-react package
+        run: rush test -t @azure/communication-react
+
+      # Push git tags
+      - name: Create and push git tags
+        run: |
+          git tag 1.0.0-alpha-${{ steps.datetime.outputs.datetime }}
+          git push --tags
+
+      # Publish package
+      - name: Package alpha packages for release
+        run: npm pack
+        working-directory: ./packages/communication-react
+
+      # Deploy npm package - this is done by uploading to Azure's SDK blob storage then triggering their partner release pipeline.
+      # More info: https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/1/Partner-Release-Pipeline
+      - name: Upload tarball to blob storage
+        uses: azure/CLI@v1
+        with:
+          inlineScript: |
+            az login --service-principal -u cfb5fe52-8bb2-4901-9056-70449fa2df96 -p ${{ secrets.AZURESDKPARTNERDROPS_SERVICE_PRINCIPAL_KEY }} --tenant 72f988bf-86f1-41af-91ab-2d7cd011db47
```

Comment:
Is it worth putting these ids inside secrets too? Or its ok for this to be public?

Reply:
They're OK to be public, but I like this idea of having them as secrets more, means all keys/ids used are updated in one place and gives them better semantic meaning. I'll update this

Reply:
Updated